wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -335,7 +336,8 @@ public final class SessionManager: NSObject, SessionManagerType {
         isUnauthenticatedTransportSessionReady: Bool = false,
         sharedUserDefaults: UserDefaults,
         minTLSVersion: String?,
-        deleteUserLogs: @escaping () -> Void
+        deleteUserLogs: @escaping () -> Void,
+        analyticsSessionConfiguration: AnalyticsSessionConfiguration? = nil
```

Comment:
I suggest to not provide a default value.
```suggestion
        analyticsSessionConfiguration: AnalyticsSessionConfiguration?
```